packages/gui/src/components/inputs/FontFamily/Input.tsx

```diff
@@ -309,46 +311,12 @@ export function FontFamilyInput({ label, value, onChange, onRemove }: Props) {
           if (['name', 'ital'].includes(k)) return null
           if (typeof v === 'string') return null
 
-          if (k === 'wdth') {
-            return (
-              <CustomAxis
-                key={k}
-                axisKey={k}
-                defaultValue={v.default}
-                value={value.fontVariationSettings ?? `'${k}' ${v.default};`}
-                onChange={(e: any) => handleCustomAxesChange(k, e)}
-                min={v.min}
-                max={v.max}
-                step={v.step}
-                label="Width"
-                sx={{ width: '100%' }}
-              />
-            )
-          }
-
-          if (k === 'wght') {
-            return (
-              <NumberInput
-                key={k}
-                value={value.fontWeight ?? v.default}
-                onChange={(newVal: number) =>
-                  onChange({ ...value, fontWeight: newVal })
-                }
-                min={v.min}
-                max={v.max}
-                step={v.step}
-                label="Font Weight"
-                sx={{ width: '100%' }}
-              />
-            )
-          }
```

Comment:
Removing this so that we instead set the weight on the `font-variation-settings` as described here https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Fonts/Variable_Fonts_Guide#weight

The caveat to this is that our `h1..h6` seem to have `font-weight: bold` set by default. This always overrides the `wght` prop on font variation settings, so it can appear as if nothing is changing.